crates/ruff_benchmark/benches/ty.rs

```diff
@@ -498,11 +498,8 @@ fn bench_project(benchmark: &ProjectBenchmark, criterion: &mut Criterion) {
         let diagnostics = result.len();
 
         assert!(
-            diagnostics > 1 && diagnostics <= max_diagnostics,
-            "Expected between {} and {} diagnostics but got {}",
-            1,
-            max_diagnostics,
-            diagnostics
+            diagnostics <= max_diagnostics,
+            "Expected <={max_diagnostics} diagnostics but got {diagnostics}"
         );
```

Comment:
we don't emti any diagnostics on this library (it's well-typed!), so this assertion didn't pass as-is as it expects at least 1 diagnostic